tests/unit/record-arrays/record-array-test.js

```diff
@@ -192,14 +192,25 @@ test('#_removeInternalModels', function(assert) {
   assert.deepEqual(content, [], 'now contains no models');
 });
 
+class FakeInternalModel {
+  constructor(record) {
+    this._record = record;
+    this.__recordArrays = null;
+  }
+
+  get _recordArrays() {
+    return this.__recordArrays;
```

Comment:
perhaps no one at the moment, but our tests often introspect state so moving it to a similar getter will let us test correctly